src/apps/an-delay/screens/DelayDetail.js

```diff
@@ -159,6 +168,28 @@ const DelayDetail = React.memo(({ delay, path, targetApp, loading }) => {
                     <LocalIdentityBadge entity={creator} />
                   </div>
                 </div>
+                {isFromSignalingProposal && (
+                  <div>
+                    <h2
+                      css={`
+                        ${textStyle('label2')};
+                        color: ${theme.surfaceContentSecondary};
+                        margin-bottom: ${2 * GU}px;
+                      `}
+                    >
+                      Reference
+                    </h2>
+                    <div
+                      css={`
+                        ${textStyle('body2')};
+                      `}
+                    >
+                      <Link href={getReference(path[0].description)} external>
```

Comment:
Let's store the result of the `getReference` function in a variable instead of calling it twice